setup.py

```diff
@@ -13,24 +12,44 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="dlc2kinematics",
-    version="0.0b1",
-    author="Mackenzie Mathis & Lab",
+    name="kinematik",
+    version="0.0.1",
+    author="Mackenzie Mathis Lab Members",
     author_email="mackenzie@post.harvard.edu",
-    description="a post-DeepLabCut module for kinematic analysis",
+    description="Library for kinematic analysis of DeepLabCut outputs",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/AdaptiveMotorControlLab/DLC2Kinematics/",
-    install_requires=['h5py>=2.7','imageio>=2.3.0','intel-openmp',
-                      'ipython~=6.0.0','ipython-genutils==0.2.0',
-                      'matplotlib','moviepy','numpy','opencv-python',
-                      'pandas','patsy','python-dateutil','pyyaml>=5.1','requests',
-                      'ruamel.yaml>=0.15','setuptools','scikit-image','scikit-learn',
-                      'scikit-kinematics','scipy','six','statsmodels','tables',
-                      'tqdm>4','wheel'],
+    url="https://github.com/AdaptiveMotorControlLab/Kinematik/",
+    install_requires=[
+        "h5py",
+        "intel-openmp",
+        "ipython",
+        "ipython-genutils",
+        "matplotlib>=3.0.3",
+        "numpy>=1.14.5",
+        "pandas",
+        "python-dateutil",
+        "pyyaml",
+        "requests",
+        "setuptools",
+        "scikit-image",
+        "scikit-learn",
+        "scikit-kinematics",
+        "scipy",
+        "tables",
+        "umap",
+        "tqdm",
+        "ruamel.yaml",
```

Comment:
ruamel.yaml>=0.15.0 (this was required for DLC as (if I remember correctly) they had introduced a new YAML parser)